11-apply-manifest-code/README.md

```diff
@@ -0,0 +1,279 @@
+# Applying Manifest Code With Bolt
+
+> **Difficulty**: Advanced
+
+> **Time**: Approximately 20 minutes
+
+In this exercise you will further explore Puppet Plans by using the `apply` keyword to leverage existing content from the [Puppet Forge](https://forge.puppet.com/).
+
+You can read more about using bolt `apply` in Masterless Workflows in a [Blog Post](https://puppet.com/blog/introducing-masterless-puppet-bolt) written by Bolt developer @MikaelSmith. 
+
+You will deploy two web servers and a load balancer to distribute the traffic evenly between them with the following steps:
+1. Build a project specific configuration using a `Boltdir`.
+1. Download useful module content from the Puppet forge. 
+1. Write a Puppet Class to abstract the configuration of an Nginx web server. 
+1. Write a Bolt Plan to `apply` puppet code and orchestrate the deployment of a static website. 
+
+# Prerequisites
+
+For the following exercises you should have `bolt` installed and have Docker and Docker-compose installed. The following guides will help:
+
+1. [Acquiring Nodes](../02-acquiring-nodes)
+1. [Writing Advanced Plans](../09-writing-advanced-plans)
+
+# Aquire nodes
+
+This lesson requires three nodes. You can use the `docker-compose.yml` file in this repository to provision the nodes necessary for this exercise. 
+
+Nodes can be obtained with the `docker-compose up -d` command.
+
+You can verify nodes are created with `docker ps`
+```
+8b7f33d8fde4        lab_node            "/usr/sbin/sshd -D"   About an hour ago   Up About an hour    0.0.0.0:20023->22/tcp                          11applymanifestcode_server_1_1
+90f6abe8dfc8        lab_node            "/usr/sbin/sshd -D"   About an hour ago   Up About an hour    0.0.0.0:20024->22/tcp                          11applymanifestcode_server_2_1
+26c47f8c4bad        lab_node            "/usr/sbin/sshd -D"   About an hour ago   Up About an hour    0.0.0.0:20022->22/tcp, 0.0.0.0:20080->80/tcp   lb
+```
+
+Alternatively you can use the vagrant nodes provisioned in [lesson 2](../02-aquiring-nodes). Remeber to configure ssh config. 
```

Comment:
`Remeber` -> `Remember`. Also `configure ssh config` sounds funny, maybe `setup ssh config`?

Reply:
I gave up trying to get the Vagrant config to work. I ran in to an issue where the load balancer just hangs when you curl/browser it. The server nodes will serve the content but the LB causes an indefinite hang.